run_scans.py

```diff
@@ -22,7 +22,7 @@ def launchTestsArgs(tool, shelf, slot, link, chamber, vfatmask, scanmin, scanmax
   #Build Commands
   setupCmds = []
   preCmd = None
-  cmd = ["%s"%(tool),"-s%d"%(slot),"-g%d"%(link),"--shelf=%i"%(shelf), "--nevts=%i"%(nevts), "--vfatmask=%s"%(vfatmask)]
+  cmd = ["%s"%(tool),"-s%i"%(slot),"-g%i"%(link),"--shelf=%i"%(shelf), "--nevts=%i"%(nevts), "--vfatmask=%i"%(vfatmask)]
```

Comment:
I had tried `--vfatmask=%x` and it didn't work (learned why after googling more about optparser class:

https://docs.python.org/2/library/optparse.html#optparse-standard-option-types

Relying on optparser to identify the type here also allows the user to enter both hex or a binary number (whereas the above does not).  Since optparser automatically handles this I don't think the change is necessary.